docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md

```diff
@@ -13,14 +13,14 @@ sidebar_position: 3
 
 The **LSP1UniversalReceiverDelegateUP** is a contract called by the **[`universalReceiver(...)`](./lsp0-erc725-account.md#universalreceiver)** function of the **[LSP0ERC725Account](./lsp0-erc725-account.md)** contract that:
 
-- Writes the keys representing **[LSP7-DigitalAsset](./lsp7-digital-asset.md)** and **[LSP8-IdentifiableDigitalAsset](./lsp8-identifiable-digital-asset.md)** assets received into the account storage, and remove them when the balance is zero according to the **[LSP5-ReceivedAssets Standard](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-5-ReceivedAssets.md)**.
+- Writes the keys representing the assets received from the **[LSP7-DigitalAsset Standard](./lsp7-digital-asset.md)** and **[LSP8-IdentifiableDigitalAsset Standard](./lsp8-identifiable-digital-asset.md)** into the account storage and removes them when the balance is zero according to the **[LSP5-ReceivedAssets Standard](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-5-ReceivedAssets.md)**.
 
-- Writes the keys representing **[LSP9-Vault](./lsp9-vault.md)** vaults owned into your account storage, and remove them when **transferring ownership** to other accounts according to the **[LSP10-ReceivedVaults Standard](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-5-ReceivedAssets.md)**.
+- Writes the keys representing the vaults owned from the **[LSP9-Vault Standard](./lsp9-vault.md)** into your account storage, and removes them when **transferring ownership** to other accounts according to the **[LSP10-ReceivedVaults Standard](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-5-ReceivedAssets.md)**.
```

Comment:
I would prefer to keep the old one with fixing the grammar mistakes because this is not right:
> the vaults owned from the **[LSP9-Vault Standard](./lsp9-vault.md)** into your account storage

The vaults are not owned from the LSP9-Vault, they are owned by the account and their type is LSP9-Vault.